raiden/blockchain_events_handler.py

```diff
@@ -32,44 +31,43 @@
 def handle_tokennetwork_new(raiden, event: Event):
     """ Handles a `TokenNetworkCreated` event. """
     data = event.event_data
-    token_network_address = data['token_network_address']
+    args = data['args']
+    block_number = data['block_number']
+    token_network_address = args['token_network_address']
+    token_address = typing.TokenAddress(args['token_address'])
 
     token_network_proxy = raiden.chain.token_network(token_network_address)
     raiden.blockchain_events.add_token_network_listener(
         token_network_proxy=token_network_proxy,
         contract_manager=raiden.contract_manager,
-        from_block=data['blockNumber'],
```

Comment:
This is bad usage, we should stick to one convention inside our code base, this key is now removed from the dict on the outermost layer, namely the function `decode_event_to_internal`